playwright.config.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+module.exports = {
+  testDir: "./test/e2e-playwright",
+  fullyParallel: true,
+  forbidOnly: !!process.env.CI,
+  retries: process.env.CI ? 2 : 0,
```

Comment:
make the CI env a const variable instead

Reply:
done

Reply:
It should be like - `const isCI = process.env.CI === 'true'` at the top, then directly use `isCI` everywhere else.

Reply:
done